Pcap++/header/PcapLiveDevice.h

```diff
@@ -186,6 +186,38 @@ namespace pcpp
 			PCPP_OUT
 		};
 
+		/**
```

Comment:
Since we are now refactoring comments to use three slashes `///` in other places, I think it's better to use `///` here directly, right? @Dimi1010

Reply:
Yes, I agree 👍